addons/cookoff/functions/fnc_engineFire.sqf

```diff
@@ -43,7 +43,7 @@ _smoke attachTo [_vehicle, _position];
 [{
     (_this select 0) params ["_vehicle", "_smoke", "_time"];
 
-    if (!alive _vehicle || {_vehicle getHitPointDamage "HitEngine" < 0.9} || {CBA_missionTime > _time}) then {
+    if (isNull _vehicle || { !alive _vehicle } || {_vehicle getHitPointDamage "HitEngine" < 0.9} || {CBA_missionTime > _time}) then {
```

Comment:
```suggestion
    if (isNull _vehicle || {!alive _vehicle} || {_vehicle getHitPointDamage "HitEngine" < 0.9} || {CBA_missionTime > _time}) then {
```
Try to avoid those spaces.